### PR TITLE
[13.0][ADD] purchase_order_type_dashboard

### PR DESCRIPTION
**Pull request by dalonsod**
Adds a dashboard for Purchase Orders based on Purchase Order Types

### CODE REVIEW

**Comment by dalonsod** [comment]
@ps-tubtim @kittiu could you review? Thanks!

File: purchase_order_type_dashboard/models/purchase_order_type.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    @api.depends(
+        "purchase_order_ids",
+        "purchase_order_ids.state",
+        "purchase_order_ids.invoice_status",
+    )
+    def _compute_po_counts(self):
+        rfq_states = ("draft", "sent", "to approve")
+        po_states = ("purchase", "done")
+        po_model = self.env["purchase.order"]
```
Comment (kittiu) [inline purchase_order_type_dashboard/models/purchase_order_type.py]:
```suggestion
        PurchaseOrder = self.env["purchase.order"]
```
Not a blocker, but I think Odoo has adopt to use camel case for object declaration like this.

File: purchase_order_type_dashboard/views/view_purchase_order_type.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    <record id="view_purchase_order_type_kanban" model="ir.ui.view">
+        <field
+            name="name"
+        >purchase.order.type.kanban (in purchase_order_type_dashboard)</field>
+        <field name="model">purchase.order.type</field>
+        <field name="arch" type="xml">
+            <kanban class="oe_background_grey o_kanban_dashboard" create="0">
+                <field name="name" />
+                <!-- field name="journal_id" / -->
```
Comment (kittiu) [inline purchase_order_type_dashboard/views/view_purchase_order_type.xml]: Should remove this?

File: purchase_order_type_dashboard/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "depends": ["purchase_order_type"],
+    "website": "https://github.com/OCA/purchase-workflow",
+    "data": [
+        "views/view_purchase_order.xml",
+        "views/view_purchase_order_type.xml",
+        "views/menu_purchase_order.xml",
+    ],
+    "installable": True,
+    "auto_install": False,
+}
```
Comment (kittiu) [inline purchase_order_type_dashboard/__manifest__.py:20]: Would you add "mantainers" ?

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
Overall, LGTM.

image

Some comment. When I first see it, I a bit confuse between (1) and (2). Then, I know that (1) is only RFQ and (2) is only confirmed Order.

Can you add tooltip (title) to (1) and (2) to explain more, it should help user. Thanks!

**Comment by dalonsod** [comment]
@kittiu comments attended

**Comment by dalonsod** [comment]
@newtratip could you review? Thanks!

**Comment by dalonsod** [comment]
Hello @dreispt , this PR started merging three days ago and didn't finish yet. Is that right, or maybe there's any problem with it? Thanks!

**Comment by dalonsod** [comment]
> Hello @dreispt , this PR started merging three days ago and didn't finish yet. Is that right, or maybe there's any problem with it? Thanks!

Hello @simahawk @pedrobaeza could you take a look at this? Thanks!

**Pull request merged**